public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
A to potrzebne? Ja bym wywalil narazie wszystkie te ikonki

Reply:
wywalone